ManageWikiExtensions.php

```diff
@@ -272,6 +272,9 @@
 			],
 		],
 		'install' => [
+			'mwscript' => [
+				'$IP/extensions/MirahezeMagic/maintenance/createCargoDB.php' => [],
```

Comment:
To make the style more consistent with the rest of extensions that run maintenance scripts, please use double quotes in this string.

Reply:
Woops that's not even a styling thing, pretty sure that's required for `$IP` to work.